basic_bot:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  


  joint1_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm1_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
  joint2_position_controller:
    type: effort_controllers/JointPositionController
    joint: arm2_joint
    pid: {p: 100.0, i: 0.01, d: 10.0}
